modules/compiler/vecz/source/pass.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include <cstdlib>
 #include <functional>
+#include <optional>
```

Comment:
Is this needed? We're adding `#include <optional>` to vecz/pass.h which is being included by this file.

Reply:
whoops sorry, already merged. It's not technically required but clang-tidy has started auto-including headers to avoid relying transitive dependencies. I guess making things more stable at the expense of compile time? I'm not sure about it, personally.

Reply:
Oh sorry it might be clangd: https://clangd.llvm.org/guides/include-cleaner#scenarios-and-solutions